src/main/java/nextstep/member/ui/MemberController.java

```diff
@@ -41,8 +43,9 @@ public ResponseEntity<MemberResponse> deleteMember(@PathVariable Long id) {
     }
 
     @GetMapping("/members/me")
-    public ResponseEntity<MemberResponse> findMemberOfMine() {
-        return ResponseEntity.ok().build();
+    public ResponseEntity<MemberResponse> findMemberOfMine(@AuthenticationPrincipal UserPrincipal userPrincipal) {
+        MemberResponse memberResponse = memberService.findMemberByEmail(userPrincipal.getUsername());
```

Comment:
email을 통해 멤버를 조회하는데 getter 메서드명을 알맞게 설정하면 좋을 것 같아요!
```suggestion
        MemberResponse memberResponse = memberService.findMemberByEmail(userPrincipal.getEmail());
```